app/views/categories/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="navigation py-6 px-3 blue-nav-white-text">
+  <ul class="flex justify-between items-center blue-btn w-full">
+    <li class="w-10"><%= link_to image_tag('backarrow.png', alt: 'Back'), categories_path, class: 'nav-link' %></li>
+    <h2>Create A Category</h2>
+    <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" %></li>
```

Comment:
Kindly fix the layout of the `new category` form page, and `transactions` page.

#### New category form

- The input fields and button are  perfectly visible. Kindly make sure that the content is perfectly aligned on the page.

![image](https://github.com/iamsjunaid/budget-bee/assets/38088818/fce0c502-3b4e-438e-8c80-27b4bc0b261e)

#### Transaction page

- At the top of the page the total amount for the category should be tubular similar to the category card with the category name on the left and price on the right.
- Kindly make sure that the transactions tab fills 100% of the mobile view width with the transaction name on the left and price on the right. This card must also be tabular.

You should have something similar to this, although in this case you do not have to match 100% of the design:

![image](https://github.com/iamsjunaid/budget-bee/assets/38088818/4c0b218a-6b78-4872-8c82-372869cc4aaf)

Your transaction page:

![image](https://github.com/iamsjunaid/budget-bee/assets/38088818/0719dc83-475a-4069-b742-ce885d759128)